test/e2e/logging_test.go

```diff
@@ -0,0 +1,102 @@
+)
+
+// TestDebugLogLevel sets 'Debug' LogLevel on the console operator and tests
+// if '--log-level=*=DEBUG' flag is set on the console deployment
+func TestDebugLogLevel(t *testing.T) {
+	client := testframework.MustNewClientset(t, nil)
+	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	testframework.MustManageConsole(t, client)
+	testframework.MustNormalLogLevel(t, client)
+
+	err := testframework.SetLogLevel(t, client, operatorsv1.Debug)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+
+	deployment, err := testframework.GetConsoleDeployment(client)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+	flagToTest := getFlag(operatorsv1.Debug)
+	if !isFlagInCommand(t, deployment.Spec.Template.Spec.Containers[0].Command, flagToTest) {
+		t.Fatalf("error: flag not found in command %v \n", deployment.Spec.Template.Spec.Containers[0].Command)
+	}
+}
+
+// TestTraceLogLevel sets 'Trace' LogLevel on the console operator and tests
+// if '--log-level=*=TRACE' flag is set on the console deployment
+func TestTraceLogLevel(t *testing.T) {
+	client := testframework.MustNewClientset(t, nil)
+	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	testframework.MustManageConsole(t, client)
+	testframework.MustNormalLogLevel(t, client)
+
+	err := testframework.SetLogLevel(t, client, operatorsv1.Trace)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+
+	deployment, err := testframework.GetConsoleDeployment(client)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+	flagToTest := getFlag(operatorsv1.Trace)
+	if !isFlagInCommand(t, deployment.Spec.Template.Spec.Containers[0].Command, flagToTest) {
+		t.Fatalf("error: flag not found in command %v \n", deployment.Spec.Template.Spec.Containers[0].Command)
+	}
+}
+
+// TestTraceLogLevel sets 'TraceAll' LogLevel on the console operator and tests
+// if '--log-level=*=TRACE' flag is set on the console deployment
+func TestTraceAllLogLevel(t *testing.T) {
+	client := testframework.MustNewClientset(t, nil)
+	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	testframework.MustManageConsole(t, client)
+	testframework.MustNormalLogLevel(t, client)
+
+	err := testframework.SetLogLevel(t, client, operatorsv1.TraceAll)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+
+	deployment, err := testframework.GetConsoleDeployment(client)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+	flagToTest := getFlag(operatorsv1.TraceAll)
+	if !isFlagInCommand(t, deployment.Spec.Template.Spec.Containers[0].Command, flagToTest) {
+		t.Fatalf("error: flag not found in command %v \n", deployment.Spec.Template.Spec.Containers[0].Command)
+	}
+}
+
+func isFlagInCommand(t *testing.T, command []string, loggingFlag string) bool {
+	t.Logf("checking if '%s' flag is set on the console deployment container command...", loggingFlag)
+	for _, flag := range command {
+		if flag == loggingFlag {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
Perhaps we should use a map to clarify the `1:1` relationship between the key & value here: 

```golang
type LogMapping map[string]string

var logMap = LogMapping{
   operatorsv1.Normal: "--log-level=*=NOTICE",
   operatorsv1.Debug: "--log-level=*=DEBUG",
   operatorsv1.Trace: "--log-level=*=TRACE",
   operatorsv1.TraceAll: "--log-level=*=TRACE",
}
```
Might even be cleaner to use in the `deployment.go` file & shared here.  Thoughts?